js/stringProcessing/cleanText.js

```diff
@@ -0,0 +1,53 @@
+/** @module stringProcessing/cleanText */
+
+var stripSpaces = require( "../stringProcessing/stripSpaces.js" );
+var replaceDiacritics = require( "../stringProcessing/replaceDiacritics.js" );
+var unifyWhitespace = require( "../stringProcessing/unifyWhitespace.js" );
+
+/**
+ * Removes words, duplicate spaces and sentence terminators, and words consisting of only digits
+ * from the text. This is used for the flesh reading ease test.
+ *
+ * @param {String} text The text to clean
+ * @returns {String} The text
+ */
+module.exports = function( text ) {
+	if ( text !== "" ) {
```

Comment:
Here we would benefit from an early return, because it reduces the nesting level of the whole function.